client/src/components/tasks/TaskForm.vue

```diff
@@ -16,14 +16,6 @@
         label="Assignee"
         outlined
       ></v-select>
-      <v-select
```

Comment:
All changes that are in the client directory are probably not intended, are they? Are you sure that your branch is up to date with the master branch and that all changes already in master are also correctly reflected in this branch? If I were to accept and merge this PR, I feel that I would als revert some features that already are in the master branch. This should not happen.